daprdocs/content/en/operations/observability/tracing/dash0.md

```diff
@@ -0,0 +1,205 @@
+
+## Prerequisites
+
+* A running Kubernetes cluster with `kubectl` installed
+* Helm v3+
+* [Dapr installed in the cluster](https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/)
+* A Dash0 account ([Get started with a 14-day free trial](https://www.dash0.com/pricing))
+* Your Dash0 **Auth Token** and **OTLP/gRPC endpoint** (find both under **Settings → Auth Tokens** and **Settings → Endpoints**)
+
+
+## Configure the OpenTelemetry Collector 
+
+1) Create a namespace for the Collector
+
+```bash
+kubectl create namespace opentelemetry
+```
+
+2) Create a Secret with your Dash0 **Auth Token** and **Endpoint**
+
+```bash
+kubectl create secret generic dash0-secrets \
+  --from-literal=dash0-authorization-token="<your_auth_token>" \
+  --from-literal=dash0-endpoint="<your_otlp_grpc_endpoint>" \
+  --namespace opentelemetry
+```
+
+3) Add the OpenTelemetry Helm repo (once)
+
+```bash
+helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+helm repo update
+```
+
+4) Create `values.yaml` for the Collector
+
+This config:
+
+* Reads token + endpoint from the Secret via env vars
+* Enables OTLP receivers (gRPC + HTTP)
+* Sends **traces, metrics, and logs** to Dash0 via OTLP/gRPC with Bearer auth
+
+```yaml
+mode: deployment
+fullnameOverride: otel-collector
+replicaCount: 1
+
+image:
+  repository: otel/opentelemetry-collector-k8s
+
+extraEnvs:
+  - name: DASH0_AUTHORIZATION_TOKEN
+    valueFrom:
+      secretKeyRef:
+        name: dash0-secrets
+        key: dash0-authorization-token
+  - name: DASH0_ENDPOINT
+    valueFrom:
+      secretKeyRef:
+        name: dash0-secrets
+        key: dash0-endpoint
+
+config:
+  receivers:
+    otlp:
+      protocols:
+        grpc: {}
+        http: {}
+
+  processors:
+    batch: {}
+
+  exporters:
+    otlp/dash0:
+      auth:
+        authenticator: bearertokenauth/dash0
+      endpoint: ${env:DASH0_ENDPOINT}
+
+  extensions:
+    bearertokenauth/dash0:
+      scheme: Bearer
+      token: ${env:DASH0_AUTHORIZATION_TOKEN}
+    health_check: {}
+
+  service:
+    extensions:
+      - bearertokenauth/dash0
+      - health_check
+    pipelines:
+      traces:
+        receivers: [otlp]
+        processors: [batch]
+        exporters: [otlp/dash0]
+      metrics:
+        receivers: [otlp]
+        processors: [batch]
+        exporters: [otlp/dash0]
+      logs:
+        receivers: [otlp]
+        processors: [batch]
+        exporters: [otlp/dash0]
+```
+
+5) Install/upgrade the Collector with Helm
+
+```bash
+helm upgrade --install otel-collector open-telemetry/opentelemetry-collector \
+  --namespace opentelemetry \
+  -f values.yaml
+```
+
+## Configure Dapr to send telemetry to the Collector
+
+1) Create a configuration
+
+Create `dapr-config.yaml`:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: tracing
+  namespace: default
```

Comment:
This configuration is deployed to `default`, but you created a namespace for `opentelemetry` so does this work? or should the namespace be in the endpoint address below, which is more likely, since this configuration is in the application namespace (i.e. default)?

Reply:
@msfussell Yep, that should work - the endpoint address takes the namespace into account 👍